### PR TITLE
test/e2e/framework:remove TODO

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**

/kind cleanup
see https://github.com/kubernetes/kubernetes/pull/88370/files#r387186433
/cc @oomichi 

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

ref:https://github.com/kubernetes/kubernetes/issues/86763

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: test/e2e/framework/network/utils.go [hunk 1]
```diff
@@ -336,8 +336,6 @@ func (config *NetworkingTestConfig) GetEndpointsFromContainer(protocol, containe
 func (config *NetworkingTestConfig) DialFromNode(protocol, targetIP string, targetPort, maxTries, minTries int, expectedEps sets.String) {
 	var cmd string
 	if protocol == "udp" {
-		// TODO: It would be enough to pass 1s+epsilon to timeout, but unfortunately
```
Comment (tanjunchen) [inline test/e2e/framework/network/utils.go]: now , it seems that we use the https://github.com/kubernetes/kubernetes/pull/88370/files#diff-85e02e1f07138a061f89e5550f36290fL84 image。

**Comment by oomichi** [comment]
Thanks for doing this.

/retest
/lgtm

**Comment by tanjunchen** [comment]
kindly /cc @neolit123

**Comment by neolit123** [comment]
> now , it seems that we use the https://github.com/kubernetes/kubernetes/pull/88370/files#diff-85e02e1f07138a061f89e5550f36290fL84 image。

/approve

**Comment by tanjunchen** [comment]
@neolit123 could you help me tag it ? thanks

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Comment by neolit123** [comment]
> Not mergeable. Must be in milestone v1.18.

it's code freeze so this can merge later this month.

**Comment by tanjunchen** [comment]
> > Not mergeable. Must be in milestone v1.18.
> 
> it's code freeze so this can merge later this month.

done

**Pull request merged**